src/presence/presence-handler.ts

```diff
@@ -310,4 +312,11 @@ export class PresenceHandler {
       })
     }
   }
+
+  private onConnectionLost (): void {
```

Comment:
here you also need to call `queryEmitter.off()` and `queryAllEmitter.off()` right?